lib/html/template/rose-bush/suites.html

```diff
@@ -11,87 +11,111 @@
 </head>
 <body>
 
-<div class="navbar navbar-static-top">
-  <div class="navbar-inner">
-    <a class="brand" href="{{script}}/">
-      {% if logo -%}
+<!-- Static navbar -->
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed"
+      data-toggle="collapse" data-target="#navbar" aria-expanded="false"
+      aria-controls="navbar">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="{{script}}/">
+        {% if logo -%}
         <img {{logo}} />
-      {% endif -%}
-      {{title}} <small>@ {{host}}</small>
-    </a>
-    <ul class="nav">
-      <li class="active"><a><strong>{{user}}</strong></a></li>
-    </ul>
-    <ul class="nav pull-right">
-      <li>
-        {{entries|length}} installed suite(s)
-        <span class="label">
-          <abbr class="livestamp" title="{{time}}">{{time}}</abbr>
-        </span>
-      </li>
-    </ul>
+        {% endif -%}
+        {{title}} <small>@ {{host}}</small>
+      </a>
+    </div>
+    <div id="navbar" class="navbar-collapse collapse">
+      <ul class="nav navbar-nav">
+        <li class="active"><a href="#"><strong>{{user}}</strong></a></li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li>
```

Comment:
If you enclose it all in a `<span class="navbar-text>` then it will vertically align nicely